CHANGELOG.md

```diff
@@ -10,7 +10,9 @@
 
 #### Enhancements
 
-* None.
+* Speedup LetVarWhitespaceRule.  
```

Comment:
Rather than claim a speed up here, can we instead just mention that you fixed a false positive? OSSCheck doesn’t demonstrate a genuine performant improvement, which may be a factor of its overhead but still.